diff --git a/chrome/common/media/cdm_host_file_path.cc b/chrome/common/media/cdm_host_file_path.cc
index a454dd3450ae06949ab38f2d854b7a9f811604ff..1bb000892c5ce19238c72d53f46b5c3fb4faeaf8 100644
--- a/chrome/common/media/cdm_host_file_path.cc
+++ b/chrome/common/media/cdm_host_file_path.cc
@@ -18,7 +18,7 @@
 #include "chrome/common/chrome_constants.h"
 #endif
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(BRAVE_CHROMIUM_BUILD)
 
 namespace {
 
@@ -44,7 +44,11 @@ void AddCdmHostFilePaths(
 #if defined(OS_WIN)
 
   static const base::FilePath::CharType* const kUnversionedFiles[] = {
+#if defined(BRAVE_CHROMIUM_BUILD)
+      FILE_PATH_LITERAL("brave.exe")};
+#else
       FILE_PATH_LITERAL("chrome.exe")};
+#endif
   static const base::FilePath::CharType* const kVersionedFiles[] = {
       FILE_PATH_LITERAL("chrome.dll"), FILE_PATH_LITERAL("chrome_child.dll")};
 
