diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index cb8e5c0c34fe0b909221d01b87aa74c99fd731be..6e24d38b62ece8c748802a5e851ce1114f3aacfc 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -175,6 +175,7 @@ jumbo_component("modules") {
     "//third_party/webrtc_overrides:task_queue_factory",
     "//third_party/zlib",
   ]
+  deps += [ "//brave/third_party/blink/renderer" ]
 
   public_deps = sub_modules
   allow_circular_includes_from = sub_modules
